src/search/move_picker.h

```diff
@@ -342,15 +396,27 @@ class MovePicker {
             }
         }
         if (bestMove.has_value())
-            moves.nullifyMove(bestMoveIndex);
+            m_moves.nullifyMove(bestMoveIndex);
 
         return bestMove;
     }
 
     SearchTables& m_searchTables;
+
     uint8_t m_ply {};
+    PickerPhase m_phase { PickerPhase::TtMove };
     std::optional<movegen::Move> m_ttMove { std::nullopt };
     std::optional<movegen::Move> m_prevMove { std::nullopt };
-    PickerPhase m_phase { PickerPhase::TtMove };
+
+    movegen::ValidMoves m_moves {};
+
+    static syzygy::WdlResult s_wdl;
+    static uint8_t s_dtz;
```

Comment:
if these are static then we should probably store them in the syzygy implementation. Doesn't feel right to have them in the move picker :)

Reply:
Moved this over :) Going forwards, I think a cleaner solution would be to convert the syzygy namespace to a class, and have a static object that owns the result data. Follow up?